taskcluster/ci/build-apk/kind.yml

```diff
@@ -38,26 +39,31 @@ task-defaults:
         max-run-time: 7200
         chain-of-trust: true
 
+
 tasks:
     focus-debug:
+        attributes:
+            code-review: true
         description: 'Focus debug build from source code'
         run-on-tasks-for: [github-pull-request, github-push]
         run:
             gradle-build-type: debug
             gradle-build-name: focusDebug
             gradle-build: focus
         treeherder:
-            symbol: debug(Bf)
+            symbol: focus-debug(Bf)
```

Comment:
Good call! I agree. Moreover, if we want to be in line with what we have in Gecko, we should use a Treeherder platform here rather than a group. I filed [bug 1805383](https://bugzilla.mozilla.org/show_bug.cgi?id=1805383) to keep track of it.